app/src/main/java/ch/blinkenlights/android/vanilla/SharedPrefHelper.java

```diff
@@ -0,0 +1,28 @@
+package ch.blinkenlights.android.vanilla;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.preference.PreferenceManager;
+
+public final class SharedPrefHelper {
+
+	/**
+	 * Cached app-wide SharedPreferences instance.
+	 */
+	private static SharedPreferences sSettings;
+
+	private SharedPrefHelper(){};
+
+	/**
+	 * Return the SharedPreferences instance containing the PlaybackService
+	 * settings, creating it if necessary.
```

Comment:
Given that this doesn't live in playback service anymore, please change the text to:

...instance containing the Application settings, ....

Reply:
I tested the settings instance can be retrieved from SharedPrefManager even when close app and stop service I press on play in widget in home the setting retrieved normally.